backend/src/main/java/com/bakdata/conquery/io/result/ResultUtil.java

```diff
@@ -54,11 +54,11 @@ public static Charset determineCharset(String userAgent, String queryCharset) {
 				return Charset.forName(queryCharset);
 			}
 			catch (Exception e) {
-				log.warn("Unable to map '{}' to a charset. Defaulting to UTF-8", queryCharset);
+				log.warn("Unable to map '{}' to a charset.", queryCharset);
 			}
 		}
 		if (userAgent != null && userAgent.toLowerCase().contains("windows")) {
-			return StandardCharsets.ISO_8859_1;
+			return Charset.forName("WINDOWS-1252");
```

Comment:
Du könntest den Lookup sparen, aber ich habe das CharSet noch nicht als static irgendwo gesehen
```suggestion
			return new sun.nio.cs.MS1252();
```